dev/gpctl/cmd/public-api-workspaces-list.go

```diff
@@ -19,19 +18,18 @@ var publicApiWorkspacesListCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		organizationID := args[0]
 
-		httpClient, address, opts, err := newConnectHttpClient()
+		conn, err := newPublicAPIConn()
```

Comment:
FWIW, I'd get rid of the sub-command entirely. I don't think it's been used really